### PR TITLE
Update VMS

### PR DESCRIPTION
**Pull request by paktek123**

### CODE REVIEW

**Review by SirMuSharif — APPROVED** [review]
Good Job

File: VMS [hunk 1]
```diff
@@ -1 +1,2 @@
 New Vm
+Hi Mustafa
```
Comment (SirMuSharif) [inline VMS]: Plz Change this asap

**Review by SirMuSharif — CHANGES_REQUESTED** [review]

File: VMS (hunk 1 above)
Comment (paktek123) [reply VMS]: ok mustafa what do you want me to change it to

Comment (SirMuSharif) [reply VMS]: the naming

Comment (paktek123) [reply VMS]: ok I will change it to my name instead

File: VMS [hunk 2]
```diff
@@ -1 +1,2 @@
 New Vm
+Hi Neeran
```
Comment (SirMuSharif) [inline VMS:2]: Great

**Review by SirMuSharif — APPROVED** [review]

**Pull request merged**